### ISSUE TITLE
Add a default die-on value to the spec metadata

### ISSUE THREAD

**Issue by jyasskin** [speced/bikeshed#2623]
As mentioned in https://github.com/whatwg/meta/issues/286#issuecomment-1648799950, it'd be great for specs to be able to include their default [`--die-on`](https://speced.github.io/bikeshed/#cli-options) value in their [metadata](https://speced.github.io/bikeshed/#metadata). The fact that a spec is warning-clean is a property of the specification, so every build shouldn't have to remember to pass that argument.

**Comment by tabatkins** [comment]
Yeah, that sounds reasonable. It's a *little bit* of a layering violation since you can get errors before I even successfully parse the metadata, but that's fine, I'll just have to check if any errors *have* been emitted after metadata parsing is complete.

**Comment by inexorabletash** [comment]
It looks like maybe this happened in 82444e6692d9717c5dc936149f8497ff67cfbf19

... but this issue isn't closed, and the docs weren't updated. Is there more to be done?

**Comment by tabatkins** [comment]
I need to fix up the docs, is all.
